### PR TITLE
Basic Login Button Completed

### PR DESCRIPTION
**Pull request by darshan3**
We will have to add contact no. cross verification to make this secure. This is just basic implementation.

### CODE REVIEW

File: app/src/main/java/com/example/darknight/mi2016/LoginActivity.java [hunk 1]
```diff
@@ -3,40 +3,141 @@
...
+        try {
+            intent.putExtra("CITY", Jobject.getString("CITY"));
+        } catch (JSONException e) {
+            e.printStackTrace();
+        }
+        try {
+            intent.putExtra("COLLEGE", Jobject.getString("COLLEGE"));
+        } catch (JSONException e) {
+            e.printStackTrace();
+        }
```
Comment (unstablebrainiac) [inline app/src/main/java/com/example/darknight/mi2016/LoginActivity.java:208]: Is there a reason to not have a single `try-catch` block?

File: app/src/main/java/com/example/darknight/mi2016/LoginActivity.java [hunk 2]
```diff
@@ -3,40 +3,141 @@
...
+                mi_no.setVisibility(View.VISIBLE);
+                submit_button.setVisibility(View.VISIBLE);
+            }
+        });
+
+        submit_button.setOnClickListener(new View.OnClickListener() {
+            public void onClick(View v) {
+
+                String mi_no_text = mi_no.getText().toString().toUpperCase();
+                Pattern pattern = Pattern.compile("MI-[A-Z]{3}-[0-9]{3,4}");
```
Comment (unstablebrainiac) [inline app/src/main/java/com/example/darknight/mi2016/LoginActivity.java]: MI-ABC-000 should not be a valid MI number

**Review by unstablebrainiac — CHANGES_REQUESTED** [review]
MI number regex can be improved and multiple try-catch blocks can be merged into one

File: app/src/main/java/com/example/darknight/mi2016/LoginActivity.java (hunk 2 above)
Comment (arunabh98) [reply app/src/main/java/com/example/darknight/mi2016/LoginActivity.java]: Are you sure ki 000 would not be a valid MI number. Some people have special MI numbers such as 007, 001 etc. Some may have 000 at the end?

**Review by arunabh98 — APPROVED** [review]
Apart from the changes which Sajal has requested, this looks good to me!!

File: app/src/main/java/com/example/darknight/mi2016/LoginActivity.java (hunk 2 above)
Comment (darshan3) [reply app/src/main/java/com/example/darknight/mi2016/LoginActivity.java]: As arunabh said, due to this special cases, I left it like this. I have currently not considered case of wrong MI number (null http response).I will add it in next commit.

Comment (arunabh98) [reply app/src/main/java/com/example/darknight/mi2016/LoginActivity.java]: Haan, wahi. If the MI number is not there in the database, just implement that toast only, I think

**Pull request merged**